docs/resiliency/disaster-recovery-azure-applications.md

```diff
@@ -36,31 +36,31 @@ For more information on how Azure Traffic Manager works, refer to:
+Another option is to store data in an alternate location until connectivity is restored. If degrading functionality is not an option, the remaining options are application downtime or failover to an alternate region. The design of an application running with degraded functionality is as much a business decision as a technical one. This is discussed further in the section on [degraded application functionality](#degraded-application-functionality).
 
 ### Failure of a dependent service
-Azure provides many services that can experience periodic downtime. Consider [Azure Redis Cache](https://azure.microsoft.com/services/cache/) as an example. This multi-tenant service provides caching capabilities to your application. It's important to consider what happens in your application if the dependent service is unavailable. In many ways, this scenario is similar to the network outage scenario. However, considering each service independently results in potential improvements to your overall plan.
+Azure provides many services that can experience periodic downtime. For example, [Azure Redis Cache](https://azure.microsoft.com/services/cache/) is a multi-tenant service which provides caching capabilities to your application. It's important to consider what happens in your application if the dependent service is unavailable. In many ways, this scenario is similar to the network outage scenario. However, considering each service independently results in potential improvements to your overall plan.
 
-Azure Redis Cache provides caching to your application from within your cloud service deployment, which provides disaster recovery benefits. First, the service now runs on roles that are local to your deployment. Therefore, you're better able to monitor and manage the status of the cache as part of your overall management processes for the cloud service. This type of caching also exposes new features. One of the new features is high availability for cached data. This helps to preserve cached data if a single node fails by maintaining duplicate copies on other nodes.
+Azure Redis Cache provides caching to your application from within your cloud service deployment, which provides disaster recovery benefits. First, the service now runs on roles that are local to your deployment. Therefore, you're better able to monitor and manage the status of the cache as part of your overall management processes for the cloud service. This type of caching also exposes new features such as high availability for cached data, which preserves cached data if a single node fails by maintaining duplicate copies on other nodes.
```

Comment:
This paragraph isn't correct, Redis Cache does not run locally in a Cloud Service role. Looking at the history, this was originally about Azure Shared Caching and someone updated it to Redis Cache without updating any of the technical details.